source/guides/best-practices.md

```diff
@@ -158,7 +158,7 @@ On NixOS, `$NIX_PATH` can be set permanently with the [`nix.nixPath`](https://se
 
 To quickly obtain packages for demonstration, we use the following concise pattern:
 
-```nix
+```nix skip
```

Comment:
Since you're only testing nix-repl blocks, there should be no need to spam every other block with redundant information (the purpose of which is not evident without "just knowing" what it means).